change/@fluentui-react-f74bc323-4380-4898-a854-205fb9b839f3.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix(Calendar): Calendar should memoize default value since it's a new Date() to avoid rerenders.",
```

Comment:
```suggestion
  "comment": "fix(Calendar): Calendar should memoize today's default value since it causes rerenders by creating a new object each time.",
```